src/components/ImageSlider/index.js

```diff
@@ -83,44 +141,127 @@ class Images extends Component {
     }
 
     let scrollViewStyles = { paddingBottom }
-
+    if (!arrows) return <View></View>
```

Comment:
This is not a viable solution. We should have `defaultProps` on `ImageSlider` for `arrows: {}`,